### PR TITLE
Only use id in create-article name if defined

### PR DESCRIPTION
**Pull request by nlisgo**
Currently the `schema:name` is set to `Article undefined` if `createArticle()` is called.

### CODE REVIEW

File: test/create-article.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Iri, JsonLdObj } from 'jsonld/jsonld-spec';
 
 const isNotUndefined = (arg: unknown): boolean => arg !== undefined;
 
-export default (id?: Iri, name = `Article ${id}`): JsonLdObj => (
+export default (id?: Iri, name = id ? `Article ${id}` : 'Article'): JsonLdObj => (
```
Comment (giorgiosironi) [inline test/create-article.ts:6]: would expect a corresponding test that fails before the change and passes after the change?

Comment (giorgiosironi) [reply test/create-article.ts:6]: (assuming this is the correct behavior)

Comment (nlisgo) [reply test/create-article.ts:6]: I had considered adding a `create-article.test.ts`.

Comment (nlisgo) [reply test/create-article.ts:6]: I have added a failing test now and followed up with the fix.

**Comment by nlisgo** [comment]
https://github.com/libero/article-store/pull/89/commits/b1f8fe7b8637133aed23de97d11cb50c08ea8859 should have failed.

https://github.com/libero/article-store/runs/336662411

**Comment by giorgiosironi** [comment]
Weird, does it reproduce locally?

**Comment by giorgiosironi** [comment]
b1f8fe7b8637133aed23de97d11cb50c08ea8859 does fail for me locally.

**Comment by giorgiosironi** [comment]
The [Travis CI build](https://travis-ci.com/libero/article-store/builds/139694115) points to a [merge commit](https://github.com/libero/article-store/commit/0d0b907685f8f95ab6b67b5489c7b3cb40c271ed).

**Comment by giorgiosironi** [comment]
Even though I can diff that commit with `master`, can't download it or open a PR with it. I think it's gone for good.

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**